ovs/extensions/db/arakoon/ArakoonInstaller.py

```diff
@@ -630,7 +630,8 @@ def _deploy(config, filesystem, offline_nodes=None):
             ServiceManager.add_service(base_name, root_client,
                                        params={'CLUSTER': config.cluster_id,
                                                'NODE_ID': node.name,
-                                               'CONFIG_PATH': config_path},
+                                               'CONFIG_PATH': config_path,
+                                               'ALBA_VERSION_CMD': "alba=`alba version --terse`" if config.export()['global']['plugins'] else ""},
```

Comment:
This command should be passed in where the plugins are passed in as well. E.g. a tuple with the plugin path and the "extra version stuff"